pkg/apis/abac/register.go

```diff
@@ -26,15 +26,27 @@ import (
 const Group = "abac.authorization.kubernetes.io"
 
 // Scheme is the default instance of runtime.Scheme to which types in the abac API group are registered.
+// TODO: remove this, abac should not have its own scheme.
```

Comment:
Looks like we have the same issue in `federation/apis/core/register.go` and `plugin/pkg/scheduler/api/register.go`